tests/unit/test_quant.py

```diff
@@ -29,36 +28,33 @@ def random_docs():
         yield d
 
 
-def get_output(req):
-    np.random.seed(531)
-
-    err = 0
-    for d in req.docs:
-        for c in d.chunks:
-            recv = pb2array(c.embedding)
-            send = np.random.random([embed_dim])
-            err += np.sum(np.abs(recv - send)) / embed_dim
-
-    print(f'reconstruction error: {err / num_docs:.6f}')
+def test_quant():
+    def get_output(req):
+        np.random.seed(531)
 
+        err = 0
+        for d in req.docs:
+            for c in d.chunks:
+                recv = pb2array(c.embedding)
+                send = np.random.random([embed_dim])
+                err += np.sum(np.abs(recv - send)) / embed_dim
 
-class MyTestCase(JinaTestCase):
+        print(f'reconstruction error: {err / num_docs:.6f}')
```

Comment:
should this stay?

Reply:
I would think no, but I want to not affect, and handle this in a different one